### DISCUSSION TITLE
Welcome to Trader_2.0 Discussions!

### DISCUSSION THREAD

**Discussion by sbtoonz** [sbtoonz/Trader_2.0#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PaloAlo** [comment]
hi, i made up a trader file with all the default vanilla stuff in it. with prices i set which i feel are reasonable and vanilla based.

i will just put it here in case you want to use it with your next release. i had to rename it to ./txt or it wouldnt let me upload it. 

[trader_config.txt](https://github.com/sbtoonz/Trader_2.0/files/10737806/trader_config.txt)

**Reply by sbtoonz** [reply]
> hi, i made up a trader file with all the default vanilla stuff in it. with prices i set which i feel are reasonable and vanilla based.
> 
> i will just put it here in case you want to use it with your next release. i had to rename it to ./txt or it wouldnt let me upload it.
> 
> [trader_config.txt](https://github.com/sbtoonz/Trader_2.0/files/10737806/trader_config.txt)

included in latest release as EXAMPLE_trader_config.yml
